gpflow/config/__config__.py

```diff
@@ -76,7 +76,7 @@ class _Values(enum.Enum):
     INT = np.int32
     FLOAT = np.float64
     POSITIVE_BIJECTOR = "softplus"
-    POSITIVE_MINIMUM = None
+    POSITIVE_MINIMUM = 0.0
```

Comment:
was this a bug? it ssems like a bit thing to change from None to 0.

Reply:
I don't know what the motivation behind initializing it as None was originally - @awav do you remember? Setting it to `None` had the same effect as 0.0 - not adding a shift to the exp/softplus transform. But it meant having to do extra queries on checking whether default_positive_minimum() was None or a float. And the setter only allowed you to set it to a float, not back to None. So it seemed more sensible to tidy it up altogether.

Reply:
None is easier to check and it is safer "label". Comparing with `0.0` is always a gamble in the world of multiple float types. I would stick to the `None`. But, please be careful with `0.0` if you decide to leave it.

@jameshensman, @st--